### PR TITLE
Currency Icon Color Gradients

### PR DESCRIPTION
**Pull request by samholmes**
### CHANGELOG

Does this branch warrant an entry to the CHANGELOG?

- [ ] Yes
- [x] No

### Dependencies

 none

### Requirements

If you have made **any** visual changes to the GUI. Make sure you have:

- [ ] Tested on iOS device
- [ ] Tested on Android device
- [ ] Tested on small-screen device (iPod Touch)
- [ ] Tested on large-screen device (tablet)

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1206061725717169

### CODE REVIEW

File: yarn.lock
Comment (swansontec) [inline yarn.lock:1]: Run `yarn-deduplicate` on this file.

File: android/settings.gradle [hunk 1]
```diff
@@ -2,3 +2,6 @@ rootProject.name = 'co.edgesecure.app'
 apply from: file("../node_modules/@react-native-community/cli-platform-android/native_modules.gradle"); applyNativeModulesSettingsGradle(settings)
 include ':app'
 includeBuild('../node_modules/react-native-gradle-plugin')
+
+apply from: new File(["node", "--print", "require.resolve('expo/package.json')"].execute(null, rootDir).text.trim(), "../scripts/autolinking.gradle")
+useExpoModules()
```
Comment (swansontec) [inline android/settings.gradle:7]: Missing newline at the end here.

File: src/components/ui4/CryptoIconUi4.tsx [hunk 2]
```diff
@@ -115,6 +125,31 @@ const CryptoIconComponent = (props: Props) => {
...
+        if (colors.platform === 'ios') {
+          console.log(colors)
+          onIconColor(colors.primary)
+        }
+        if (colors.platform === 'android') {
+          onIconColor(colors.dominant)
+        }
+      })
+      .catch(err => {
+        console.error(err)
```
Comment (swansontec) [inline src/components/ui4/CryptoIconUi4.tsx:149]: `console.error` gives the red screen of doom. Please use `console.log` or `console.warn` instead.

File: src/components/ui4/CryptoIconUi4.tsx [hunk 3]
```diff
@@ -115,6 +125,31 @@ const CryptoIconComponent = (props: Props) => {
...
+  React.useEffect(() => {
+    if (primaryCurrencyIconUrl == null || onIconColor == null) return
+
+    getColors(primaryCurrencyIconUrl, {
+      cache: true,
+      key: primaryCurrencyIconUrl
+    })
+      .then(colors => {
+        if (colors.platform === 'ios') {
+          console.log(colors)
```
Comment (swansontec) [inline src/components/ui4/CryptoIconUi4.tsx:141]: Leftover logging?

File: src/components/scenes/TransactionListScene.tsx [hunk 4]
```diff
@@ -230,9 +234,10 @@ function TransactionListComponent(props: Props) {
   })
 
   return (
-    <NotificationSceneWrapper navigation={navigation} hasTabs scroll>
+    <NotificationSceneWrapper navigation={navigation} hasTabs>
```
Comment (swansontec) [inline src/components/scenes/TransactionListScene.tsx:237]: Was removing the `scroll` intentional? Not a problem if so - just double-checking.

**Review by swansontec — APPROVED** [review]
This seems fine, aside from a few tiny things.

Now that we are in the Expo boat, we can consider using more libraries from this ecosystem, such as their gradient view. I get the impression that these libraries may be better-maintained than some of the random NPM stuff we are using.

File: src/components/scenes/TransactionListScene.tsx (hunk 4 above)
Comment (samholmes) [reply src/components/scenes/TransactionListScene.tsx:237]: It was. We should have the FlashList be the scroll

**Pull request merged**